indirect.h

```diff
@@ -351,26 +351,26 @@ struct std::hash<xyz::indirect<T, Alloc>> {
 
 namespace xyz {
 
-template <class T>
-concept is_formattable = requires(T t) { std::formatter<T, char>{}; };
-    
-}
+template <class T, class charT>
+concept is_formattable = requires(T t) { std::formatter<T, charT>{}; };
```

Comment:
Could be:

```
#if (__cpp_lib_format_ranges >= 202207L)

template <class T, class charT>
concept formattable = std::formattable<T, charT>;

#else

template <class T, class charT>
concept formattable = requires { std::formatter<T, charT>{}; };

#endif  // (__cpp_lib_format_ranges >= 202207L)
```

I've tested this on Visual Studio and Clang 17.  Even though the [paper that introduced `std::formattable` used the `__cpp_lib_format` feature test flag](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2022/p2286r8.html#feature-test-macro) it looks like [a subsequent LWG issues was raised to change this to `__cpp_lib_format_ranges` due to to many concurrent papers bumping this feature macro](https://cplusplus.github.io/LWG/issue3750)

Reply:
Lets merge and fix later where needed.